next-cloudinary/src/lib/cloudinary.ts

```diff
@@ -32,7 +32,7 @@ let cld;
  * constructCloudinaryUrl
  */
 
-export function constructCloudinaryUrl({ options, config }) {
+export function constructCloudinaryUrl({ options, config }: { options?: any; config?: any }): string {
```

Comment:
Doesn't the Cloudinary SDK have types for `options` and `config` already that we can just plug in here?

Reply:
it 100% obvious to me without a deeper dive if this code was _only_ using the cloudinary config options. The answer is: Probably, but could also wait until this is merged in to be done as an incremental improvement

Reply:
Again, a sensible approach.

The [cloudinary](https://github.com/cloudinary/cloudinary_npm) package has types that might be useful here at some point.